src/components/FeeExplainer/FeeSliders.tsx

```diff
@@ -100,9 +144,21 @@ export const FeeSliders: React.FC<FeeSlidersProps> = ({
         </Stack>
       </Stack>
       <Stack width='full' spacing={4}>
-        <Flex width='full' justifyContent='space-between' fontWeight='medium'>
+        <Flex width='full' justifyContent='space-between' alignItems='center' fontWeight='medium'>
           <Text translation={featureSizeTranslation} />
-          <Amount.Fiat fiatType='USD' value={tradeSizeUSD} fontWeight='bold' />
+          <Box sx={inputStyle} width='35%'>
+            <NumberFormat
+              decimalScale={2}
+              customInput={Input}
+              isNumericString={true}
+              prefix={'$'}
```

Comment:
```suggestion
              suffix={localeParts.postfix : ''}
              prefix={localeParts.prefix : ''}
```

Reply:
No, FeeExplainer is using USD only

Reply:
Changed in [6b12ae3](https://github.com/shapeshift/web/pull/7984/commits/6b12ae3b57f16c7a37fc2699a84daf2c62b3147f) but it's wrong as we are using $, if our users change it to something else than USD, the prefix will use the "something else" currency but keep the same number as $, because initially we didn't want to use those settings for simplicity